src/components/ChallengeModal/ChallengeModal.test.tsx

```diff
@@ -403,44 +212,6 @@ describe("ChallengeModalBody", () => {
         expect(maxRankSelect).toBeInTheDocument();
     });
 
-    it.skip("renders AI player options when in computer mode", () => {
```

Comment:
I wasn't sure what the plan for this one was.  Just deleting bc it is getting skipped, but I can re-add if there were plans to light it up again.